.pipelines/singletenancy/cilium-nodesubnet/cilium-nodesubnet-e2e-step-template.yaml

```diff
@@ -0,0 +1,196 @@
+parameters:
+  name: ""
+  clusterName: ""
+  scaleup: ""
+
+steps:
+  - task: AzureCLI@2
+    displayName: 'Update IP configs'
```

Comment:
Instead of bash script can this be done via golang ?

Reply:
I have an implementation of this in Python too actually. The tricky thing is, we need the AzureCLI@2 task in order to get access to the cluster. The AzureCLI task seems to allow only Bash/Powershell scripts, not Python/Golang. Now, we could get around that by invoking Python/Go in the bash script, I wasn't sure if we wanted that extra level of invocation. From what I understand, we don't use Python in our pipelines at all today, not sure about Go. Honestly, I'm not fond of this bash script myself, I'd much rather use a proper programming language for this.

Reply:
Thinking out loud here: Can't we add a make command in the acn repo that takes in the network profile from the az cli commands. and return the updated network profile (using the go lang).
Keep using the az cli command as it is, just instead of bash we have a make command that does the manipulation(in golang)

Reply:
We can. I'd actually prefer for the whole thing to be in Go, not just the calculation of the new profile. But invoking Go from Bash is the extra layer of invocation I was talking about. Let me just go ahead and do it.